1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,214 +1,191 @@
-# Comparisons
+# तुलना
 
-We know many comparison operators from maths.
+हम गणित से कई तुलना ऑपरेटरों को जानते हैं।
 
-In JavaScript they are written like this:
+जावास्क्रिप्ट में वे इस तरह लिखे जाते हैं:
 
-- Greater/less than: <code>a &gt; b</code>, <code>a &lt; b</code>.
-- Greater/less than or equals: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
-- Equals: `a == b`, please note the double equality sign `==` means the equality test, while a single one `a = b` means an assignment.
-- Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as <code>a != b</code>.
+- ग्रेटर / से कम: <code>a &gt; b</code>, <code>a &lt; b</code>।
```

Comment:
We can refer to the link above for some of these notations. Whichever notations are not available (in Hindi) in the list, you can translate them by using an online translator, Use words and provide some context to get more accurate results.

Reply:
@theBstar which link? Can you help me with this, I have added all other changes that were requested